charts/vector/values.yaml

```diff
@@ -0,0 +1,48 @@
+indexPattern: "%Y.%m.%d"
+
+nodeSelector: {}
+affinity: {}
+tolerations: []
+
+podSecurityContext: {}
+
+vector:
+  securityContext:
+    runAsUser: 0
+  image:
+    repository: quay.io/astronomer/ap-vector
+    tag: 0.47.0-4
```

Comment:
In this PR we need to update `bin/show-docker-images.py` to show this image.

Also, we will need to modify bom-generator to work with this.